.github/workflows/publish-packages.yaml

```diff
@@ -15,10 +15,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/workflows/env-setup
-      - name: Create Release Pull Request
+      - name: Create Release Pull Request/Publish to npm
         uses: changesets/action@v1
         with: 
           version: pnpm ci:version
+          publish: pnpm ci:publish
```

Comment:
I thought the changeset action would publish by default with `changeset publish`, but it does require an explicit command